### PR TITLE
Add webgroup/blockbot job

### PR DESCRIPTION
**Pull request by JedHazaymeh**
Add jobspec for "blockbot" container

- [Dockerfile](https://github.com/redbrick/blockbot/blob/main/Dockerfile)
- [GitHub Action](https://github.com/redbrick/blockbot/blob/main/.github/workflows/build.yml)

### CODE REVIEW

File: jobs/webgroup/blockbot.hcl [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  group "blockbot" {
+    count = 1
+
+    task "blockbot" {
+      driver = "docker"
+
+      config {
+        image = "ghcr.io/redbrick/blockbot"
+        entrypoint = ["python3", "src/main.py"]
```
Comment (DistroByte) [inline jobs/webgroup/blockbot.hcl:14]: This should really be defined in the Dockerfile

**Review by DistroByte — COMMENTED** [review]
One small comment, aside from that all good

File: jobs/webgroup/blockbot.hcl (hunk 1 above)
Comment (JedHazaymeh) [reply jobs/webgroup/blockbot.hcl:14]: It is, so all good 👍 I'll remove the line

**Review by wizzdom — APPROVED** [review]
tested successfully on aperture

**Pull request merged**